TapOrHoldCounter/MainVC.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MainVC.swift
+//  TapOrHoldCounter
+//
+//  Created by vmalikov on 1/28/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+class MainVC: UIViewController {
+
+    @IBOutlet weak var counter: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        resetTapped(self)
+    }
+    
+    @IBAction func resetTapped(sender: AnyObject) {
```

Comment:
I usually try to specify the object in outlet names or action names. In this case without any background you don't know what reset is. It might be a label with tap recognizer, an image, whatever. But if it will be `resetButtonTapped` there is more context right there: this is a button and it is tapped (most likely from inside out - that's the most common case).